src/app/config.service.ts

```diff
@@ -28,9 +28,19 @@ export class ConfigService {
    *
    * @return {boolean}
    */
-  useHttpdServer(): boolean {
-    const value = get(this.config, 'useHttpdServer');
-    return isBoolean(value) ? value : value === 'true';
+  useHttpServer(): boolean {
+    const value = get(this.config, 'contentSearch.enable');
+    return isBoolean(value) ? value : value == 'true';
```

Comment:
No!
**===** is a strict comparison, so true is not equal to 'true' and you have to implement your solution
**==** is a soft comparison, so true is equal to 'true'. JS will convert the data before to compare them.

In you case, you can simplify the code like this:
```javascript
return value == true;
```